### PR TITLE
Added method to delete pods based on namespace and label

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
1)Added method to delete pods based on namespace and label
2)Corrected the formatting 
**Which issue(s) this PR fixes** (optional)
Closes #
https://portworx.atlassian.net/browse/PA-577
**Special notes for your reviewer**:

### CODE REVIEW

File: tests/backup/backup_helper.go [hunk 1]
```diff
@@ -84,6 +81,10 @@ var backupAccessKeyValue = map[BackupAccess]string{
 	3: "FullAccess",
 }
 
+var StorkLabel = map[string]string{
```
Comment (mkoppal-px) [inline tests/backup/backup_helper.go:84]: Make it `storkLabel`

Comment (snigdha-px) [reply tests/backup/backup_helper.go]: done

**Comment by mkoppal-px** [comment]
Can we run these 2 tests and make sure it works fine - `KillStorkWithBackupsAndRestoresInProgress` and `RestartBackupPodDuringBackupSharing`

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Comment by snigdha-px** [comment]
> Can we run these 2 tests and make sure it works fine - `KillStorkWithBackupsAndRestoresInProgress` and `RestartBackupPodDuringBackupSharing`

https://jenkins.pwx.dev.purestorage.com/job/Users/job/Sumit/job/Custom-Pipelines/job/px-backup-on-demand-system-test/192/consoleFull

**Review by mkoppal-px — APPROVED** [review]
LGTM

File: tests/backup/backup_helper.go [hunk 2]
```diff
@@ -1233,3 +1234,22 @@ func GetAllRestoresNonAdminCtx(ctx context.Context) ([]string, error) {
 	}
 	return restoreNames, nil
 }
+
+// KillPodWithLabelInGivenNamespace kills pod with the given label in the given namespace
+func KillPodWithLabelInGivenNamespace(namespace string, label map[string]string) error {
```
[the remark this answers is not in the document]
Comment (snigdha-px) [reply tests/backup/backup_helper.go]: done

**Pull request merged**